src/Examine.Test/Examine.Lucene/Search/FluentApiTests.cs

```diff
@@ -117,21 +207,121 @@ public void Uppercase_Category()
 
                 Console.WriteLine(query);
 
-                var results = query.Execute();
+
+                var results = query.WithFacets(facets => facets.Facet("nodeName")).Execute();
+
+                var facetResults = results.GetFacet("nodeName");
+
+                Assert.AreEqual(3, results.TotalItemCount);
+                Assert.AreEqual(3, facetResults.Count());
+            }
+        }
+
+        [Test]
+        public void FacetsConfig_SetIndexName_Long()
+        {
+            var fieldDefinitionCollection = new FieldDefinitionCollection(new FieldDefinition("parentID", FieldDefinitionTypes.Integer), new FieldDefinition("LongValue", FieldDefinitionTypes.FacetLong));
+
+            var facetsConfig = new FacetsConfig();
+            facetsConfig.SetIndexFieldName("LongValue", "facet_longvalue");
```

Comment:
This test scenario should not occur. SetIndexFieldName is only meant to be called when there is a separate Taxonomy index in use. See the Lucene.NET docs for this method.